content/doc/book/system-administration/administering-jenkins.adoc

```diff
@@ -0,0 +1,135 @@
+
+* On Windows By default, this is set to `C:\ProgramData\Jenkins\.jenkins`.
+* On Ubuntu By default, this is set to `~/.jenkins`.
+
+But you can change this in one of the following ways:
+
+* Set the `JENKINS_HOME` environment variable
+to the location of the new home directory
+before launching the servlet container.
+* Set the `JENKINS_HOME` system property in the servlet container.
+* Set the `JNDI` environment entry `JENKINS_HOME` to the new directory.
+See the link:/doc/book/installing/index[Installing Jenkins] for details about how to do this for your container.
+
+You can change this location after you've used Jenkins for a while, too.
+To do this:
+
+. stop Jenkins completely.
+. Move the contents from the old `JENKINS_HOME` to the new location.
+. Set the `JENKINS_HOME` variable to the new location.
+. Restart Jenkins.
+
+The directory structure of the `JENKINS_HOME` tree is structured as follows:
+[width="100%",cols="100%",]
+|===
+a|
+....
+JENKINS_HOME
+ +- config.xml     (Jenkins root configuration file)
+ +- *.xml          (other site-wide configuration files)
+ +- identity.key   (RSA key pair that identifies an instance)
+ +- secret.key     (deprecated key used for some plugins' secure operations)
+ +- secret.key.not-so-secret  (used for validating _$JENKINS_HOME_ creation date)
+ +- userContent    (files served under your https://server/userContent/)
+ +- secrets        (root directory for the secret+key for credential decryption)
+     +- hudson.util.Secret   (used for encrypting some Jenkins data)
+     +- master.key           (used for encrypting the hudson.util.Secret key)
+     +- InstanceIdentity.KEY (used to identity this instance)
+ +- fingerprints   (stores fingerprint records, if any)
+ +- plugins        (root directory for all Jenkins plugins)
+     +- [PLUGINNAME]   (sub directory for each plugin)
+         +- META-INF       (subdirectory for plugin manifest + pom.xml)
+         +- WEB-INF        (subdirectory for plugin jar(s) and licenses.xml)
+     +- [PLUGINNAME].jpi   (.jpi or .hpi file for the plugin)
+ +- jobs           (root directory for all Jenkins jobs)
+     +- [JOBNAME]      (sub directory for each job)
+         +- config.xml     (job configuration file)
+         +- workspace      (working directory for the version control system)
+         +- latest         (symbolic link to the last successful build)
+         +- builds         (stores past build records)
+             +- [BUILD_ID]     (subdirectory for each build)
+                 +- build.xml      (build result summary)
+                 +- log            (log file)
+                 +- changelog.xml  (change log)
+     +- [FOLDERNAME]   (sub directory for each folder)
+         +- config.xml     (folder configuration file)
+         +- jobs           (subdirectory for all nested jobs)
+....
+|===
+
+=== Backup and restore
+link:/doc/book/system-administration/backing-up/[Backing-up/Restoring Jenkins ]
+
```

Comment:
The backup and restore section is already well covered.  Delete from this pull request
```suggestion
```